src/php/Formatter/Formatter/FormatterInterface.php

```diff
@@ -2,7 +2,7 @@
 
 declare(strict_types=1);
 
-namespace Phel\Formatter;
+namespace Phel\Formatter\Formatter;
```

Comment:
I don't like these duplicated namespaces like,
`Formatter\Formatter`
`Compiler\Compiler`

Do you have any better ideas?

Reply:
Good point. Let's start defining what does this means for each "level" in the namespace:
- The first `Formatter` is actually the module. 
- The second `Formatter` is a directory inside the module (we could also call it a "submodule", but that's not relevant here).

Being aware of this, the directory "Formatter" inside the module "Formatter" might seems a bit boilerplate, but it's in favor of the pattern that every other module is also having in place. Because the main point here is that at the module-root-level we'll find only the module stuff (Facade, Factory, Config, DependencyProvider). And from here, we can simply organize the submodules/directories following a simply DDD pattern. This is why I named it that way "Formatter\Formatter". The domain language of the module will group (and name) the actual logic of it.

I would keep it for now as it is, but if we came with other ideas we can always apply them in another iteration. I am always open to suggestions :)